package.json

```diff
@@ -3,14 +3,16 @@
   "description": "WebXR Device API Specification",
   "version": "0.0.1",
   "devDependencies": {
-    "browser-sync": "^2.18.8"
+    "browser-sync": "^2.18.8",
+    "doctoc": "^1.4.0"
   },
   "scripts": {
     "start": "npm run dev",
```

Comment:
Unrelated to this PR, but.. what is this for?  Perhaps I'm doing it wrong, but I've been opening the file directly in a browser after the build?

Reply:
'browser-sync'?  I thought you added it! 😨